public/src/news.js

```diff
@@ -0,0 +1,59 @@
+const app = document.getElementById('root')
+const newlist = document.querySelector('.new-list')
+
+const logo = document.createElement('img')
+logo.src='images/logo.png'
+
+const container = document.createElement('div')
+container.setAttribute('class', 'container')
+
+
+app.appendChild(logo)
+app.appendChild(container)
+
+
+fetch('https://content.guardianapis.com/search?show-blocks=all&api-key=3d1bd118-3d6f-4211-a61e-3d0f966b08bb')
+
+.then(response => {
+  return response.json()
+  
+})
+.then(data => {
+    data.response.results.forEach(news => {
+     var li = document.createElement('li')
+     var a = document.createElement('a')
+     a.setAttribute('href', onclick=retrieve)
+     a.setAttribute('target', '_blank')
+     a.text = "see article"
+     const card = document.createElement('div')
+     card.setAttribute('class', 'card')
+     const h3 = document.createElement('h3')
+     h3.textContent = news.webTitle
+     li.appendChild(a)
+     container.appendChild(card)
+     card.appendChild(h3)
+     card.append(li)
+     window.webUrl=news.webUrl
```

Comment:
Not sure what you are trying to do with this line. Right now, what you are doing is creating a global variable called `webUrl` and writing the piece of news URL in it for each piece of news. Which means in the end you have overwritten it a number of times.

Reply:
You are right, it doesn't work properly. 
Here, creating global webUrl and then the retrieve function takes this Url. Otherwise, i can't transfer the url to the outside function... AHA!! or this is the time i am gonna use callback. ?????
Every time it takes same Url and user clicks the see article, it shows same summarized article.